incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -0,0 +1,580 @@
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.command.dump.internal.airline;
+
+import static java.lang.Integer.parseInt;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.agrona.DirectBuffer;
+import org.agrona.LangUtil;
+import org.agrona.MutableDirectBuffer;
+import org.agrona.collections.LongHashSet;
+import org.agrona.concurrent.BackoffIdleStrategy;
+import org.agrona.concurrent.IdleStrategy;
+import org.agrona.concurrent.UnsafeBuffer;
+
+import com.github.rvesse.airline.annotations.Command;
+import com.github.rvesse.airline.annotations.Option;
+
+import io.aklivity.zilla.runtime.command.ZillaCommand;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.labels.LabelManager;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.layouts.StreamsLayout;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.spy.RingBufferSpy;
+import io.aklivity.zilla.runtime.command.dump.internal.types.Flyweight;
+import io.aklivity.zilla.runtime.command.dump.internal.types.PcapGlobalHeaderFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.PcapPacketHeaderFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.TcpHeaderFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.AbortFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.BeginFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.ChallengeFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.DataFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.EndFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.FlushFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.FrameFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.ResetFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.SignalFW;
+import io.aklivity.zilla.runtime.command.dump.internal.types.stream.WindowFW;
+
+@Command(name = "dump", description = "Dump stream content")
+public final class ZillaDumpCommand extends ZillaCommand
+{
+    @Option(name = {"-v", "--verbose"},
+        description = "Show verbose output")
+    public boolean verbose;
+
+    @Option(name = {"-b", "--bindingNames"},
+        description = "Dump specific namespaces, bindings only, e.g example.http0,example.kafka0")
+    public List<String> bindingNames = new ArrayList<>();
```

Comment:
Even though this contains "binding names" we should just call it `bindings` as there is no other interpretation for the string command line argument and keeps it simpler plus easier to type.